web/src/utils/crypto.test.ts

```
@@ -0,0 +1,55 @@
+import {
+  fulfillKeyRequest,
+  storeReceivedSessionKey,
+  getMyKeyPair,
+  decryptMessage,
+  encryptMessage,
+} from './crypto';
+import * as keychainDb from '@lib/keychainDb';
+import * as socket from '@lib/socket';
+import * as keyManagement from '@utils/keyManagement';
+
+import { describe, it, expect, vi, afterEach } from 'vitest';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'describe'. Consider to delete it or rename it.

'describe' is defined here:


(lint/suspicious/noRedeclare)